fire/cli/niv/__init__.py

```diff
@@ -500,16 +500,79 @@ def find_sagsid(sagsgang: pd.DataFrame) -> str:
 def punkter_geojson(
     projektnavn: str,
     punkter: pd.DataFrame,
+    præfix: str = "",
```

Comment:
Ikke for at være pedantisk, da det ikke gør nogen teknisk forskel, men formelt set burde det vel hedde infiks istedet for præfiks, da det bliver sat inde i tekststrengen og ikke foran:)

Reply:
Jeg kunne ikke lige finde på et bedre navn så det blev bare præfix. Lad os bare køre med `infiks`